ui-speedspacechart/src/components/helpers/drawElements/curve.ts

```diff
@@ -1,16 +1,33 @@
 import { clearCanvas, maxPositionValues, speedRangeValues } from '../../utils';
-import type { Store } from '../../../types/chartTypes';
 import { MARGINS } from '../../const';
+import type { DrawFunctionParams, LayerData } from '../../../types/chartTypes';
 
 const { CURVE_MARGIN_TOP, CURVE_MARGIN_SIDES } = MARGINS;
 
-export const drawCurve = (
-  ctx: CanvasRenderingContext2D,
+const drawSpecificCurve = (
+  specificSpeeds: LayerData<number>[],
+  minSpeed: number,
+  speedRange: number,
+  maxPosition: number,
+  ratioX: number,
   width: number,
   height: number,
-  store: Store
+  ctx: CanvasRenderingContext2D
```

Comment:
I find that the function has too many parameters. In this case, I propose to group parameters :
- Parameters related to curves (`minSpeed`, `speedRange`, `maxPosition`, `ratioX`) are grouped into an object `curveConfig`.
- Parameters related to the canvas (`width`, `height`, `ctx`) are grouped into an object `canvasConfig`.
- Parameter `specificSpeeds` can be the third parameter.

We can define new types for  `CurveConfig` et `CanvasConfig`, (May be can be used in another layers ?

Reply:
yes good point

Reply:
done